packages/gatsby-theme-parliament/src/components/MDXFilter.js

```diff
@@ -203,9 +206,9 @@ export default ({ children, pageContext }) => {
               <View>
                 {pageContext.frontmatter.contributors && (
                   <Contributors
-                    href="#"
-                    contributors={pageContext.frontmatter.contributors}
```

Comment:
AFAIK a requirement was to be able to add external contributors so we might need to extend the github contributors with the contributors defined in frontmatter ?

Reply:
This was not properly defined for all possible cases. AFAIK it was only a question in the Design XD. I'll bring it up.